examples/mixd_to_nutils.py

```diff
@@ -0,0 +1,63 @@
+import gustaf as gus
+import numpy as np
+
+
+def main():
+    # Creates a gustaf volume.
+    mesh = create_mesh()
+
+    # Export it as .mixd-file and .npz-file
+    gus.io.mixd.export(mesh, "export/export_mixd.xns")
+    gus.io.nutils.export(mesh, "export/export_npz.npz")
+
+    # Load the mixd-file and the .npz-file
+    mesh_mixd = gus.io.mixd.load(
+            volume=True,
+            mxyz="export/export_mixd.mxyz",
+            mien="export/export_mixd.mien",
+            mrng="export/export_mixd.mrng"
+    )
+    mesh_npz = gus.io.nutils.load("export/export_npz.npz")
+
+    mesh.show()
+    mesh_mixd.show()
+    mesh_npz.show()
```

Comment:
Can you show them next to each other the comparison would be easier I think. Should be something like this:

```suggestion
    gus.show([mesh, mesh_mixd, mesh_npz])
```